packages/core/src/components/tree/tree.tsx

```diff
@@ -174,22 +174,29 @@ export class Tree {
         this.updatePadding(el, item);
 
         if (!this.itemClickListener.has(el)) {
-          const itemClickCallback = (e: Event) => {
-            // Prevent hyperlist from refreshing the list to not dispose associated dropdowns
-            const hasTrigger = dropdownController.pathIncludesTrigger([el]);
+          const itemClickCallback = (event: Event) => {
+            const hasTrigger = dropdownController.pathIncludesTrigger(
+              event.composedPath()
```

Comment:
This will walk all the way to the document root. We could constrain this to the tree itself:
```
const treeIndex = event.composedPath().findIndex((el) => el === this.hostElement);
event.composedPath().slice(0, treeIndex);
```